doc/release-notes-31278.md

```diff
@@ -0,0 +1,5 @@
+Updated RPCs
+---
+- The `settxfee` RPC is now **deprecated** and will be removed in Bitcoin Core 30.0. This RPC allowed users to set a static fee rate for wallet transactions, but in a dynamic fee environment, this can lead to overpaying or underpaying. Users should instead rely on fee estimation or specify a fee rate per transaction. (#31278)
+
+As a reminder, manual fee rate selection should be done per transaction rather than globally. Fee estimation provides a more adaptive approach, and users can override it on a per-transaction basis when necessary.
```

Comment:
The last line here mostly repeats the previous sentence. Perhaps this could be deduplicated.